src/controllers/overseas.entity.query.controller.ts

```diff
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../utils/logger";
+import * as config from "../config";
+import { getApplicationData, setExtraData } from "../utils/application.data";
+import { OeNumber } from "../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.OVERSEAS_ENTITY_QUERY_PAGE}`);
+
+    return res.render(config.OVERSEAS_ENTITY_QUERY_PAGE, {
+      backLinkUrl: config.UPDATE_LANDING_PAGE_URL,
+      templateName: config.OVERSEAS_ENTITY_QUERY_PAGE
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.OVERSEAS_ENTITY_QUERY_PAGE}`);
+    const oeNumber = req.body[OeNumber];
+
+    if ( oeNumber.length === 8 ) {
```

Comment:
Known to start with "OE". Should we check that here? Also need user feedback for <> 8.

Reply:
I've actually just removed the validation stuff as Jitin (@jkayyala) was saying he has figured out the correct way to do it and will raise it separately to this ticket